vllm/worker/vineyard_llm_cache.py

```diff
@@ -193,9 +215,19 @@ def prefetch_seq_kv_caches(
             # torch.distributed.broadcast(slot_mapping, src=0,
             #                             group=get_tensor_model_parallel_group())
 
-
+        self.metrics.hit_tokens += matched
+        self.metrics.total_tokens += len(tokens)
+        self.metrics.hit_blocks += (matched // block_size)
+        self.metrics.total_blocks += (- len(tokens) // (-block_size))
+        self.metrics.counter += 1
         # save to GPU kv cache
+        torch.cuda.synchronize()
```

Comment:
what's the usage of line 224? do you want to eliminate other synchronization?  I think 227 synchronize is good enough?